hera_pspec/tests/test_pspecdata.py

```diff
@@ -2188,8 +2188,8 @@ def test_window_funcs():
 
 def test_get_argparser():
     args = pspecdata.get_pspec_run_argparser()
-    a = args.parse_args([['foo'], 'bar', '--dset_pairs', '0 0, 1 1', '--pol_pairs', 'xx xx, yy yy',
-                         '--spw_ranges', '300 400, 600 800', '--blpairs', '24 25 24 25, 37 38 37 38'])
+    a = args.parse_args([['foo'], 'bar', '--dset_pairs', '0~0,1~1', '--pol_pairs', 'xx~xx,yy~yy',
```

Comment:
Let's keep the old test with the spaces to ensure backwards compatibility, as well as the new test with the updated delimiter character

Reply:
Good idea.